app/controllers/contact_infos_controller.rb

```diff
@@ -14,8 +12,9 @@ class ContactInfosController < ApplicationController
 
   def create
     handle_with(ContactInfosCreate,
-                success: lambda {
+                success: lambda do
                   contact_info = @handler_result.outputs.contact_info
+                  security_log :contact_info_created, contact_info_id: contact_info.id
```

Comment:
for contact info create and destroy, can we record the value too? (the ID alone won't be enough if we're auditing what happened to a destroyed contact info)